packages/epo-react-lib/src/form/SelectListbox/SelectListbox.tsx

```diff
@@ -0,0 +1,141 @@
+import { FunctionComponent, useLayoutEffect, useState } from "react";
+import { useUID } from "react-uid";
+import { useTranslation } from "react-i18next";
+import { onChangeCallback, Option } from "@/types/select-listbox";
+import IconComposer from "@/svg/IconComposer";
+import * as Styled from "./styles";
+import { useAccessibleDropdown, useOnOutsideClick } from "./hooks";
+
+interface SelectListboxProps {
+  value: string | string[] | null;
+  options: Option[];
+  onChangeCallback: onChangeCallback;
+  isDisabled?: boolean;
+  placeholder?: string;
+  labelledById?: string;
+  namespace?: string;
+  isMultiselect?: boolean;
+  maxWidth?: string;
+  width?: string;
+}
+
+const SelectListbox: FunctionComponent<SelectListboxProps> = ({
```

Comment:
this is heavily derived from [this post](https://medium.com/lego-engineering/building-accessible-select-component-in-react-b61dbdf5122f) with the only major changes being implementation of a multiselect and a different click-outside setup.